cli/src/main/kotlin/StartCommand.kt

```diff
@@ -23,6 +23,7 @@ import com.github.ajalt.clikt.command.SuspendingCliktCommand
 import com.github.ajalt.clikt.parameters.options.flag
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.file
```

Comment:
Commit message: This is a breaking change.
Please use a `!` to indicate this.

```
feat(cli)!: Accept CreateOrtRun object in run command
```

You should also mention that you replace the current options with the JSON option.